src/utils/fhir.js

```diff
@@ -459,18 +487,35 @@ function getPlanDef(fullUrl) {
   else return null;
 }
 
+/**
+ * Deletes the resource from external server
+ *
+ * @param {Resource} resource
+ */
+async function deleteResource(resource) {
+  const { fullUrl } = resource;
+  if (fullUrl) {
+    const token = await getAccessToken(getBaseUrlFromFullUrl(fullUrl), db);
+    const headers = { Authorization: `Bearer ${token}` };
+    axios.delete(fullUrl, { headers });
```

Comment:
I need to load some data into my db to check what all the fullUrls are, but at a glance this generic approach to deleting feels a little scary. Not that it can get called with completely arbitrary data, it should only get called with subscriptions, but still.
As of now the only thing we want to delete on an external server is subscriptions, and that's only on the EHR server. I don't think we want to delete the subscription from the KAR, because that's generic on all PlanDefinitions. (if a subscription there even exists) So just to be safe and prevent any chance of badness, how would you feel about making this more explicit and hardcoded to Subscription resources on the EHR?

Reply:
Sounds good to me, updated the function.